app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,10 @@
+class LikesController < ApplicationController
+  def increment
+    @post = Post.find(params[:id])
+    @like = Like.create(post: @post, user: current_user)
+
+    respond_to do |format|
+      format.js { render :increment }
+    end
+  end
```

Comment:
**OPTIONAL**

- [ ] Please update your function name from `increment` to `create` and remember to update your routes